server/src/main/java/com/cloud/api/query/QueryManagerImpl.java

```diff
@@ -617,7 +618,9 @@ private Pair<List<UserAccountJoinVO>, Integer> getUserListInternal(Account calle
             ssc.addOr("email", Op.LIKE, "%" + keyword + "%");
             ssc.addOr("state", Op.LIKE, "%" + keyword + "%");
             ssc.addOr("accountName", Op.LIKE, "%" + keyword + "%");
-            ssc.addOr("accountType", Op.LIKE, "%" + keyword + "%");
+            if (EnumUtils.isValidEnum(Account.Type.class, keyword.toString().toUpperCase())) {
```

Comment:
we discussed off-line to make `keyword` a string. Will you do that @weizhouapache ?

Reply:
thanks @DaanHoogland 
done